en/docs/cli/team.md

```diff
@@ -8,8 +8,46 @@ layout: guide
 
 ##### `yarn team` <a class="toc" id="toc-yarn-team" href="#toc-yarn-team"></a>
 
-*Oops, something went wrong!*
+yarn team is a prefix used for a number of commands create destroy, add, rm and ls
```

Comment:
I don't think this is necessary to document, can just skip into the other commands